src/components/ViewRequest/StatusBar.tsx

```diff
@@ -14,7 +14,9 @@ const StatusBar = () => {
   const request = useRequest(Number(params.requestId));
 
   const stageIndex = STAGES.findIndex(({ key }) => key === request.data?.stage);
-  const currentStage = STAGES[stageIndex]?.text || "Loading...";
+  const currentStage =
+    STAGES[stageIndex]?.text ||
+    (request.data?.stage === "Canelled" ? "Cancelled" : "Loading...");
```

Comment:
I'm not sure what you are doing here, checking if it's "Canelled" (spelling), and changing the spelling if it is... otherwise it is Loading?